moduleroot/Gemfile

```diff
@@ -127,8 +139,14 @@ if explicitly_require_windows_gems
 end
 <% end -%>
 
+# Evaluate Gemfile.local if it exists
 if File.exists? "#{__FILE__}.local"
   eval(File.read("#{__FILE__}.local"), binding)
 end
 
+# Evaluate ~/.gemfile if it exists
```

Comment:
This is because when releasing a module, I run `git clean -xfd` to ensure that the puppet module build only has what it should have.

Having a gemfile.local in the repo gets obliterated.